src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write your code here
+const populationElements = document.querySelectorAll('.population');
+
+const populations = Array.from(populationElements).map((element) => {
+  // Удаляем запятые из строки населения и преобразуем в число
+  return parseInt(element.textContent.replace(/,/g, ''), 10);
+});
+
+const totalPopulation = populations.reduce((acc, curr) => acc + curr, 0);
+
+const averagePopulation = totalPopulation / populations.length;
+
+const totalPopulationElement = document.querySelector('.total-population');
+const averagePopulationElement = document.querySelector('.average-population');
+
+// Прямо вставляем числа с разделителями тысяч, используя toLocaleString()
+totalPopulationElement.innerText = totalPopulation.toLocaleString();
+
+const roundedAverage = Math.round(averagePopulation);
+
+averagePopulationElement.innerText = addThousandSeparators(roundedAverage);
```

Comment:
You're using a custom function `addThousandSeparators` to format the average population number, but you've used `toLocaleString` for the total population. It would be more consistent to use `toLocaleString` for both, as it's a built-in function that handles number localization.